nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/README.md

```diff
@@ -0,0 +1,46 @@
+# Express + OpenTelemetry example app
+
+This example Express app connects to a Cloud SQL database (Postgres or MySQL)
+using Sequelize or Knex as the ORM. It is instrumented with OpenTelemetry to
+collect trace data and send it to Cloud Trace. SQLCommenter attaches the
+OpenTelemetry trace and span ID to the queries for correlation.
+
+## Running
+
+This example reads some environment variables to connect to a Cloud SQL database.
+
+- `DBHOST` - The hostname, ip, or path to unix socket directory for Cloud SQL
+database. For example, if you are using Cloud SQL proxy with a unix socket,
+you might pass `/cloudsql/my-project:us-central1:my-db`
+- `DBUSERNAME` - Database username
+- `DBPASSWORD` - Database password
+- `DBDIALECT` - Optional. Should be `postgres` (default) or `mysql`
+
+```bash
+# in this directory
+$ export DBHOST="<hostname or unix socket path>"
+$ export DBUSERNAME="<database user>"
+$ export DBPASSWORD="<database password>"
+$ export DBDIALECT="<postgres | mysql>"
+$ npm install
+
+# on first run, create some data
+$ npm run createTodos
+
+# run the sequelize server
+$ npm run sequelizeServer
+
+# in a separate terminal
+$ curl http://localhost:8000
+{"todos":[{"id":11,"title":"Do dishes","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":12,"title":"Buy groceries","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":13,"title":"Do laundry","description":"Finish before Thursday!","done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":14,"title":"Clean room","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":15,"title":"Wash car","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.112Z","updatedAt":"2020-11-06T01:59:50.112Z"}]}
+
+# Original terminal will output
+[server]: Server is running at https://localhost:8000
+Google Cloud Trace export
+Google Cloud Trace batch writing traces
+Google Cloud Teacesauthenticating
+Google Cloud Teacesgot authentication. Initializaing rpc client
```

Comment:
What's "Google Cloud Teaces" is that output perhaps by something getting mangled up?

Reply:
Yes definitely mangled up, let me re-run it